src/desktop/package.json

```diff
@@ -136,24 +136,25 @@
     "curl.lib.js": "^1.0.22",
     "date-fns": "^1.29.0",
     "electron-settings": "^3.2.0",
-    "electron-updater": "^3.0.3",
+    "electron-updater": "^3.1.2",
     "entangled-node": "rihardsgravis/entangled-node",
-    "i18next": "^11.5.0",
+    "i18next": "^11.9.0",
+    "iota.lib.js": "^0.5.1",
```

Comment:
Isn’t 0.5.0 the latest version?

Reply:
`0.5.1` is the latest

Reply:
According to https://github.com/iotaledger/iota.js/blob/develop/package.json it's `0.5.1`

Reply:
Whoops, didn’t realize there was a new version on npm. I don’t think there was a tag for this one though